hal/inc/hal_platform.h

```diff
@@ -271,4 +271,8 @@
 #define HAL_PLATFORM_SETUP_BUTTON_UX (0)
 #endif /* HAL_PLATFORM_SETUP_BUTTON_UX */
 
+#ifndef HAL_PLATFORM_SLEEP_2_0
+#define HAL_PLATFORM_SLEEP_2_0 (0)
```

Comment:
`HAL_PLATFORM_SLEEP20` for simplicity? Less awkward typing :slightly_smiling_face: